packages/@okta/vuepress-site/docs/guides/unlock-mobile-app-with-biometrics/store-tokens/android/storerefreshtoken.md

```diff
@@ -1,3 +1,29 @@
-Example Code Note: Assume the developer's sign-in method looks like the example from: https://developer.okta.com/docs/guides/sign-into-mobile-app/ios/sign-in-page/
+First create a encryption manager and set it in the `WebAuthBuilder` like the following:
 
-Show what must be updated to store the refresh tokens with biometrics.
\ No newline at end of file
+```java
+private GuardedEncryptionManager keyguardEncryptionManager = new GuardedEncryptionManager(this, Integer.MAX_VALUE);
+...
+...
+@Override
+protected void onCreate(Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+
+    config = new OIDCConfig.Builder()
+        .clientId("{yourClientId}")
+        .redirectUri("{redirectUri}")
+        .endSessionRedirectUri("endSessionUri")
+        .scopes("openid", "profile", "offline_access")
+        .discoveryUri("{discoveryUri}")
+        .create();
+
+    webAuth = new Okta.WebAuthBuilder()
+        .withConfig(config)
+        .withContext(getApplicationContext())
+        .withEncryptionManager(keyguardEncryptionManager)
+        .create();
+        }
+}
+```
+
+This will setup the client to require the device to be authenticated in order to use the
+private key.
```

Comment:
This sets up the client to require that the device be authenticated in order to use the private key.